.devcontainer/devcontainer.json

```
@@ -10,7 +10,9 @@
     "dockerFile": "../Dockerfile",
 
     // Set *default* container specific settings.json values on container create.
-    "settings": {},
+    "settings": {
+    	"terminal.integrated.shell.linux": "/bin/bash"
```

This setting has been deprecated.
```suggestion
    "terminal.integrated.defaultProfile.linux": "bash",
```